arviz/plots/bf_plot.py

```diff
@@ -0,0 +1,83 @@
+# Plotting and reporting Bayes Factor given idata, var name, prior distribution and reference value
+import logging
+
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import stats
+# 
+_log = logging.getLogger(__name__)
+
+def plot_bf(
+    idata,
+    var_name,
+    prior,
+    ref_val=0,
+    xlim=None,
+    ax=None):
+   
+    """Bayes Factor approximated as the Savage-Dickey density ratio.
+    The Bayes factor is estimated by comparing a model 
+    against a model in which the parameter of interest has been restricted to a point-null.
+   
+    Parameters
+    -----------
+    :idata: obj
+        a :class:`arviz.InferenceData` object
+    :var_name: str 
+        Name of variable we want to test.
+    :prior: numpy.array, optional
+        In case we want to use diffent prior (for sensitivity analysis of BF), 
+        we can define one and sent it to the function.
+    :ref_val: int, default is 0
+        Reference value for BF testing
+    :xlim: numpy.array, optional
+        limit the x axis (might be used for visualization porpuses sometimes)
```

Comment:
```suggestion
    idata : obj
        a :class:`arviz.InferenceData` object
    var_name : str 
        Name of variable we want to test.
    prior : numpy.array, optional
        In case we want to use different prior, for example for sensitivity analysis.
    ref_val : int
        Point-null for Bayes factor estimation. Defaults to 0.
    xlim:  numpy.array, optional
        Set the x limits, which might be used for visualization purposes.
```